src/prune_builds

```diff
@@ -13,13 +13,27 @@ import argparse
 import subprocess
 import collections
 
-import pytz
-import dateutil.parser
-from datetime import timedelta, datetime
+from datetime import timedelta, datetime, timezone
 
 sys.path.insert(0, '/usr/lib/coreos-assembler')
 from cmdlib import write_json, rfc3339_time
 
+
+def parse_date_string(date_string):
+    """
+    Parses the date strings expected from the build system. Returned
+    datetime instances will be in utc.
+
+    :param date_string: string to turn into date. Format: %Y-%m-%dT%H:%M:%SZ
+    :type date_string: str
+    :returns: datetime instance from the date string
+    :rtype: datetime.datetime
+    :raises: ValueError, TypeError
+    """
+    dt = datetime.strptime(date_string, '%Y-%m-%dT%H:%M:%SZ')
+    return dt.replace(tzinfo=datetime.timezone.utc)
```

Comment:
This should be just `timezone.utc`, right?

Reply:
yeah, he uses `timezone.utc` later on

Reply:
Oops, yup!